shared/scripts/virtio_console_guest.py

```diff
@@ -273,7 +277,10 @@ def _get_port_status(self, in_files=None):
                         port[m.group(1)] = m.group(2)
 
                     if port['is_console'] == "yes":
-                        port["path"] = "/dev/hvc%s" % (port["console_vtermno"])
+                        port_num = int(port["console_vtermno"]) - 1 if \
```

Comment:
Better avoid use `\`, if line is too long I think we would rather use 'if ... : else: ...'

Reply:
Or cleaner
```
port_num = int(port["console_vtermno"])
if current_version >= fixed_version:
    port_num -= 1
```

Reply:
It looks better, update and thanks.